client/webserver/webserver.go

```diff
@@ -262,10 +262,10 @@ func New(core clientCore, addr string, logger dex.Logger, reloadHTML bool) (*Web
 	})
 
 	// Files
-	fileServer(mux, "/js", filepath.Join(siteDir, "dist"))
-	fileServer(mux, "/css", filepath.Join(siteDir, "dist"))
-	fileServer(mux, "/img", filepath.Join(siteDir, "src/img"))
-	fileServer(mux, "/font", filepath.Join(siteDir, "src/font"))
+	fileServer(mux, "/js", filepath.Join(siteDir, "dist"), "text/javascript")
```

Comment:
"application/javascript", right?

Reply:
Apparently not. https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types

Reply:
Also ref: https://github.com/golang/go/commit/99df76f0869f00ad9ba53e3aecc22d625ae23de1 and https://github.com/golang/go/issues/32351